storage/app/settings.dist.json

```diff
@@ -2,6 +2,11 @@
     "sync": false,
     "system.alert": false,
     "tools": [
+        {
+            "label": "Manage",
+            "route": "filament.admin.pages.dashboard",
+            "abilities": "accessManagementTools"
+        },
```

Comment:
I've noticed that my menu no longer contains any items except this one. I don't see any reason why this feature should prevent access to Logs/Horizon Queues/Mailpit/etc.
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/b7c4010c-3286-405e-94db-8ce45f71414e)

fwiw, it's blank if I don't make this change, so it's not because of this change.
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/2e009fa1-f25a-4964-9bc1-972c5f0dafae)

Reply:
forgot to add those to my todo list, thanks for the reminder! everything was visible to moderators previously because of the Gate::before hook ehich has been removed.

I'd like to make those root only, what do you think? that's the role sysadmins should have

Reply:
Maybe Architect or Engineer (not exactly sure what those imply). 

Logs and Horizon Queue are definitely geared towards the technical people, but that doesn't mean we should limit it to Root. Maybe Release Manager?

Sentry and Stats could probably be open to any Administrator, maybe even Release Manager.

Metabase should be available to Administrators and Event Managers.

Reply:
adjusted in https://github.com/RetroAchievements/RAWeb/pull/2165/commits/7c17198384c34d88478df04fc05e61c7636e5475

please change `viewLogs` to `viewLogViewer` in `storage/app/settings.json` according to the `settings.dist.json` file.

Reply:
`Architect` and `Engineer` are vanity roles and have no permissions attached.

`Release Manager` is meant to be focused on client/integration releases.

Sentry and Stats are open to admins now.

I wouldn't want to give Event Managers access to Metabase, get rid of metabase rather and build the tools needed into Filament. Metabase is becoming a liability, only meant to give tools into some people's hands until purpose-built tools are available. Too many raw queries in there...

Reply:
I think I'm still hung up on what an Administrator is. In the past, they had access to everything. It would make sense to limit these tools to that Role assuming that's the scope of the Role.

My mind would probably connect things better if the role were renamed to System Administrator or something similar.

I do see the menu items returned, but they don't seem to be affected by roles yet. I assumed they would given the fact that you had me rename `viewLogs` to `viewLogViewer`. I'm fine if you want to do that in a later PR.

Reply:
Those are all good points. It's loosely inspired by the [original permission matrix](https://docs.google.com/spreadsheets/d/17YROKCpqinnyYG3cE8Q1mLDxzOAARCFzvjGltuADZb0/edit?usp=sharing) - however, I'm not sure all of it is applicable.

We should probably discuss this with the team to figure out where our roles should be heading at.

`Root` is basically `System Administrator`. Not sure what to do about the `Administrator` role at this point. I initially planned to get rid of it altogether. That's from a time where the admin role in this community had a very bad reputation. So instead of making it a catch-all role it'd be split up into dedicated task-focused roles. Hence all the manager roles.

> I do see the menu items returned, but they don't seem to be affected by roles yet. I assumed they would given the fact that you had me rename viewLogs to viewLogViewer. I'm fine if you want to do that in a later PR.

`viewLogViewer` is the gate that the log viewer package expects itself - it's checking for the `tool` ability which is currently bound to root and admin. I'm not sure I understand correctly - in which way are they not affected by roles yet? If you mean applying role checks on individual pages, those will follow in another PR which will reduce/get rid of the usage of `UserAccounts.Permissions`.

Reply:
When my account has no roles:
```
MariaDB [retroachievements-web]> select * from auth_model_roles where model_id=43495;
Empty set (0.001 sec)
```
I still have the full menu:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/0f6a87ed-3de2-4997-85c6-e97680044c6c)

I have to drop my non-role Permissions all the way down to 1 to make them disappear - at which point the entire Manage button disappears.

Reply:
oh! that's because of the local environment check. it always displays tools locally.

Reply:
note that the legacy Permissions still is in effect, too. this makes it hard to reason about the actual permissions in some cases. a follow-up PR will hopefully resolve most of that.